http-client.ts

```diff
@@ -97,65 +95,77 @@ export class HttpClient implements Server.IHttpClient {
 
 			this.$logger.trace("httpRequest: %s", util.inspect(options));
```

Comment:
this trace must be immediately before the request - currently it will not print all options that we pass and we may be confused when asking customers for logs.